cumulus/client/cirrus-executor/src/fraud_proof.rs

```diff
@@ -78,11 +78,13 @@ where
 	pub(crate) fn generate_proof<Number, PHash>(
 		&self,
 		block_number: NumberFor<Block>,
-		local_trace_idx: usize,
+		local_trace_index: usize,
 		local_receipt: &ExecutionReceipt<Number, PHash, Block::Hash>,
 		execution_receipt: &ExecutionReceipt<Number, PHash, Block::Hash>,
 	) -> Result<FraudProof, FraudProofError> {
-		let header = self.header(execution_receipt.secondary_hash)?;
+		let block_hash = execution_receipt.secondary_hash;
```

Comment:
`secondary_hash` would be a less ambiguous name here, it is easy to forget which hash we're talking about.

Reply:
Ambiguous from the parimary hash? There is no primary chain client in `FraudProofGenerator` so it can't be the primary hash, the fraud proof generation all happen on the secondary client side, the primary client is not involved, `block_hash` matches better with `block_number` IMHO. If you still think `secondary_hash` is a better one, I don't mind changing it.

Reply:
I think it would be clearer that way, but no strong opinion right now